config/locales/client.en.yml

```diff
@@ -5001,6 +5001,8 @@ en:
         reset: "reset"
         none: "none"
       site_settings:
+        emoji_list:
+          invalid_input: "Input fields cannot contain invalid emoji name or be empty or contain vertical bar character."
```

Comment:
Maybe we can simplify this: Emoji list should only contain valid emoji names, eg: hugs